composer.json

```diff
@@ -139,7 +141,24 @@
         {
             "type": "git",
             "url": "https://github.com/mautic/intl.git"
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "mautic/grapejs-builder",
```

Comment:
```suggestion
                "name": "mautic/GrapesJsBuilderBundle",
```

Reply:
Added following snippet.
```
"extra": {
                    "installer-name": "GrapesJsBuilderBundle"
                }
```
Let's see how it goes.